.github/dependabot.yml

```diff
@@ -6,8 +6,6 @@ updates:
 # Maintain dependencies for GitHub Actions
   - package-ecosystem: "github-actions"
     directory: "/"
-    schedule:
-      interval: "daily"
```

Comment:
It seems this field is required? https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#configuration-options-for-the-dependabotyml-file

We instead may want to set `open-pull-requests-limit` to `0`:
![Screenshot 2023-11-01 at 16 19 28](https://github.com/fleetdm/fleet/assets/2073526/36d731c2-3d12-42c7-9db6-1fb94057b18a)

From https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates

Reply:
Good catch! It looks like we have that key present for some package ecosystems but not all. I've updated.